ui/src/components/banner/BannerContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+import Banner from './Banner';
+
+const BannerContainer = () => {
+  const [banners, setBanners] = useState([]);
+
+  const bannerData = process.env.REACT_APP_BANNERS;
```

Comment:
@mistryrn I understand why/how, but it seems odd seeing `process` on a browser app. lol
no action needed